pkg/models/shortUrl.go

```diff
@@ -0,0 +1,47 @@
+package models
+
+import (
+	"errors"
+)
+
+var (
+	ErrShortURLNotFound = errors.New("Short URL not found")
+)
+
+type ShortUrl struct {
+	Id         int64
+	OrgId      int64
+	Uid        string
+	Path       string
+	CreatedBy  int64
+	CreatedAt  int64
+	LastSeenAt int64
+}
+
+// ---------------------
+// COMMANDS
+
+type CreateShortURLCommand struct {
+	OrgId     int64  `json:"-"`
+	Uid       string `json:"uid"`
```

Comment:
```suggestion
	OrgID     int64  `json:"-"`
	UID       string `json:"uid"`
```

Reply:
Note to self: We actually don't want to introduce more usages of the bus in this PR, since the bus is deprecated.